packages/react/package.json

```diff
@@ -22,14 +22,14 @@
     "main": "dist/index.js"
   },
   "peerDependencies": {
-    "@spotify-confidence/sdk": "0.1.4",
+    "@spotify-confidence/sdk": "^0.1.4",
     "react": "^18.2.0"
   },
   "devDependencies": {
-    "@microsoft/api-extractor": "*",
-    "@spotify-confidence/sdk": "0.1.4",
+    "@microsoft/api-extractor": "7.43.1",
+    "@spotify-confidence/sdk": "workspace:",
```

Comment:
```suggestion
    "@spotify-confidence/sdk": "0.1.4",
```

Reply:
Fixed by pinning corresponding dev-deps to lowest possible version.